apps/service_providers/llm_service/runnables.py

```diff
@@ -452,3 +456,7 @@ def _cancel_run(self, assistant, thread_id, run_id):
             cancelling = run.status == "cancelling"
             if cancelling:
                 sleep(assistant.check_every_ms / 1000)
+
+    @property
+    def experiment(self) -> Experiment:
```

Comment:
where is this called?

Reply:
[Here](https://github.com/dimagi/open-chat-studio/pull/613/files#r1718584146)